ViAn/GUI/drawingwidget.cpp

```diff
@@ -375,6 +376,16 @@ void DrawingWidget::delete_item() {
     if (item->type() == FRAME_ITEM) {
         FrameItem* f_item = dynamic_cast<FrameItem*>(item);
         if (f_item->get_frame() == m_overlay->get_current_frame()) {
+            QMessageBox msg_box;
+            msg_box.setIcon(QMessageBox::Warning);
+            msg_box.setText("Deleting frameitem\n"
```

Comment:
`Deleting frameitem\n` should probably be renamed into something like `deleting drawings from frame` or similar since the user won't know what a frameitem is